### PR TITLE
fix: fixme line in todo action

### PR DESCRIPTION
**Pull request by G3root**
## What does this PR do?

**Environment**: Staging(main branch) / Production

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

File: .github/workflows/todo-to-issue.yml [hunk 1]
```diff
@@ -1,7 +1,6 @@
 name: TODO to Issue
 on:
   push:
```
Comment (zomars) [inline .github/workflows/todo-to-issue.yml:3]: Let's remove this line as well

Comment (G3root) [reply .github/workflows/todo-to-issue.yml:3]: you mean the line `1` ?

Comment (zomars) [reply .github/workflows/todo-to-issue.yml:3]: push:

Comment (G3root) [reply .github/workflows/todo-to-issue.yml]: pushed the changes

**Review by emrysal — APPROVED** [review]
Better late than never @zomars 🤣

**Pull request merged**